class-wds-react-post-search.php

```diff
@@ -329,7 +329,7 @@ public function post_types_to_search() {
 	 * @return void
 	 */
 	public function rest_api_init() {
-		register_rest_route('wds-react-post-search/v1', '/search', [
+		register_rest_route( 'wp/v2', '/search', [
```

Comment:
Since this is a core route now as of 5.0, I think by registering a route here, we may be conflicting with the core route. If we're going to use the core route, we will be breaking things on SIG Perks, since our filters for modifying the tax query and orderby parameters would no longer work. While I like the idea of using a core route, I think we should wait to use that until we know what we'll need to change on SIG to make it work.